crates/amaru-stores/src/rocksdb/consensus.rs

```diff
@@ -105,7 +105,7 @@ mod test {
     #[test]
     fn rocksdb_chain_store_can_get_what_it_puts() {
         let tempdir = tempfile::tempdir().unwrap();
-        let basedir = tempdir.path().join("rocksdb_chain_store");
+        let basedir = tempdir.keep().join("rocksdb_chain_store");
```

Comment:
I don't think we want `keep` here; this used to be `into_path` (now deprecated), and I recently turned it to `path`. Could it be that this change now comes from a merge-conflict resolution?

Reply:
We were using `into_path` indeed; but taking ownership wasn't needed. So `path` is sufficient.

Reply:
[into_path](https://github.com/Stebalien/tempfile/blob/95540ed3fcb9ca74845c02aee058726b2dca58b7/src/dir/mod.rs#L410) prevents the dir to be deleted automatically (as `keep` now does) so I guess the original usage wasn't expected.